pkg/prowgen/prowgen_tests.go

```diff
@@ -107,6 +108,20 @@ func DiscoverTests(r Repository, openShiftVersion string, cronOverride *string)
 			cfg.Tests = append(cfg.Tests, *cronTestConfiguration)
 		}
 
+		securityTestConfiguration := cioperatorapi.TestStepConfiguration{
+			As:       "security",
+			Optional: true,
+			MultiStageTestConfiguration: &cioperatorapi.MultiStageTestConfiguration{
+				Environment: cioperatorapi.TestEnvironment{
+					"PROJECT_NAME": r.Repo,
+					"ORG_NAME":     r.Org,
```

Comment:
Do we need to create the org before? How does it get access to it?

Reply:
Altenatively we could create a more complex project name which includes the repository org `<org>-<repo>` and maybe also branch suffix so that we have the analysis in the existing openshift org per branch instead of managing our own org, but I'm not sure what's best, I'd connect with TP team to see what they suggest and the undocumented mechanics

Reply:
I see the test run PR failure so I suspect there is something to do in this area

Reply:
Yes. I just saw that `If your organization is not OpenShift, you can optionally override the ORG_NAME environment variable to point to your organization in snyk UI.`
Have to check how to create that org and how it gets accessed

Reply:
Let's ask them in #forum-ocp-testplatform

Reply:
You already did that! :fastparrot: